helix-term/src/commands.rs

```diff
@@ -2505,17 +2505,19 @@ async fn make_format_callback(
     let format = format.await?;
     let call: job::Callback = Box::new(move |editor, _compositor| {
         let view_id = view!(editor).id;
-        if let Some(doc) = editor.document_mut(doc_id) {
-            if doc.version() == doc_version {
-                doc.apply(&format, view_id);
-                doc.append_changes_to_history(view_id);
-                doc.detect_indent_and_line_ending();
-                if let Modified::SetUnmodified = modified {
-                    doc.reset_modified();
-                }
-            } else {
-                log::info!("discarded formatting changes because the document changed");
+        let scrolloff = editor.config().scrolloff;
+        let doc = doc_mut!(editor, &doc_id);
+        let view = view_mut!(editor);
+        if doc.version() == doc_version {
+            doc.apply(&format, view_id);
```

Comment:
We can use `view.id` here and remove the `let view_id = view!(editor).id;` line above